### ISSUE TITLE
Community Guide Line (aka rule)

### ISSUE THREAD

**Issue by cryptozeny** [sugarchain-project/PM#4]
We do not have any rule on our community yet. Please suggest yours.

@aglkm
@AestheticSenpai 

For example:
https://github.com/zcash/zcash/blob/master/code_of_conduct.md

**Comment by aglkm** [comment]
I've adjusted Zcash Code of Conduct and created Sugarchain Rules. Please review.

**Sugarchain Rules**
As contributors and maintainers of this project, and in the interest of fostering an open and welcoming community, we pledge to respect all people involved in building Sugarchain eco-system.

We are committed to making participation in this project a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.

Examples of unacceptable behavior by participants include:

- The use of sexualized language or imagery
- Personal attacks
- Trolling or insulting/derogatory comments
- Public or private harassment
- Publishing other's private information, such as physical or electronic addresses, without explicit permission
- Other unethical or unprofessional conduct

Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned with Sugarchain Rules, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.

**Sugarchain Team**

**Comment by cryptozeny** [comment]
agreed. any other suggestion?

**Comment by cryptozeny** [comment]
but it seems just copy and paste from ZEC... :thinking: they are a company, but we are not.

https://electriccoin.co/about/

**Comment by aglkm** [comment]
It's a fork! :) That's a general information and it is almost the same on every community. So there is no need to invent a bicycle ;)

**Comment by cryptozeny** [comment]
> So there is no need to invent the bicycle ;)

agreed. but i think we can fork another bicycle :laughing: i will do some research
